packages/scss/src/components/indexTable/mods.scss

```diff
@@ -0,0 +1,314 @@
+        border-bottom: var(--components-indexTable-row-stack-border-bottom) solid transparent;
+        // stack apparence
+        background-image: var(--components-indexTable-row-stack2-background-image, none), var(--components-indexTable-row-stack3-background-image, none);
+        background-position: var(--components-indexTable-row-stack2-background-position, 0 0), var(--components-indexTable-row-stack3-background-position, 0 0);
+        background-repeat: no-repeat, no-repeat;
+        transition-property: background-image, background-position;
+        transition-duration: var(--commons-animations-durations-fast);
+
+        // apparence of the first cell with a card apparence.
+        // we do this because there migh be one or two cells before without background and containing a checkbox
+        --components-indexTable-row-stack2-background-position: left calc(-2px + var(--components-indexTable-cell-inset-x-left, 0px)) bottom -10px;
+        --components-indexTable-row-stack3-background-position: left calc(1px + var(--components-indexTable-cell-inset-x-left, 0px)) bottom -14px;
+
+        // apparence of the nexts cells
+        & ~ .indexTable-body-row-cell {
+            --components-indexTable-row-stack2-background-position: left -20px bottom -10px;
+            --components-indexTable-row-stack3-background-position: left -20px bottom -14px;
+        }
+
+        // apparence of the last cell with a card apparence
+        &:last-child {
+            --components-indexTable-row-stack2-background-position: right calc(-2px + var(--components-indexTable-cell-inset-x-right, 0px)) bottom -10px;
+            --components-indexTable-row-stack3-background-position: right calc(1px + var(--components-indexTable-cell-inset-x-right, 0px)) bottom -14px;
+        }
+    }
+}
+
+@mixin stickyHeader {
+    .indexTable-head {
+        position: sticky;
+        top: var(--components-indexTable-row-spacing);
+        z-index: 2;
+    }
+
+    //hide box-shadow from underneath rows that slighly appear on edges of the sticky header when scrolling
+    .indexTable-head-row-cell {
+        position: relative;
+        &::before,
+        &::after {
+            content: '';
+            position: absolute;
+            inset: calc(var(--components-indexTable-row-spacing) * -1) calc(var(--components-indexTable-padding) * -1) 0;
+            z-index: -1;
+        }
+        &::before {
+            background-color: var(--commons-background-base);
+        }
+        &::after {
+            background-color: var(--components-indexTable-background-color);
+        }
+        &:first-child::after {
+            border-top-left-radius: var(--components-indexTable-border-radius);
+        }
+        &:last-child::after {
+            border-top-right-radius: var(--components-indexTable-border-radius);
+        }
+    }
+}
+
+@mixin responsiveCardList {
+    @include media.max('S') {
```

Comment:
Bringing this MQ up to index level would enable BUs to do away with it and define it differently.